beater/server_test.go

```diff
@@ -1,4 +1,4 @@
-package server
+package beater
```

Comment:
I say it was probably ok to keep them separated, but thought you preferred to merge them. I don't have a strong preference.